walkthroughs/week-5-tdd/project/src/main/java/com/google/sps/FindMeetingQuery.java

```diff
@@ -14,10 +14,36 @@
 
 package com.google.sps;
 
+import java.util.ArrayList;
+import java.util.Arrays;
 import java.util.Collection;
+import java.util.Collections;
+import java.util.HashSet;
+import java.util.List;
+import java.util.logging.Logger;
+
 
 public final class FindMeetingQuery {
+
+  private static Logger log = Logger.getLogger("FindMeetingQuery");
```

Comment:
better to do `Logger.getLogger(FindMeetingQuery.class.getName());` (no need to create string yourself)